spec/spec_helper_acceptance.rb

```diff
@@ -38,10 +36,13 @@
       # Install module
       copy_module_to(host, :source => proj_root, :module_name => 'one')
 
+      if fact('osfamily') == 'RedHat'
+        on host, "yum -y install rubygem-nokogiri"
+        on host, "rm -rf /etc/yum.repos.d/puppetlabs.repo"
```

Comment:
Better: first yum clean all, then delete the repo file.
Otherwise things will remain broken due to the old repo still being in cache.